diagnostic.md

```diff
@@ -45,21 +49,26 @@ weather === "sunny";
 
 What are the values of these expressions?  Explain your answers.
 
+The first expression sets the weather variable to the sting value 'sunny'.  The second expression is asking to evaluate if weather is set to 'sunny', which it is, so the values of both
 
 ## Question 5
 
 ```js
 //We'll learn about require later in the course
+
 const ask = require('../lib/ask.js');
 
-let answer = 'not empty';
+let answer = 0;
 
-while (answer !== '') {
+while (answer !== false) {
   answer = ask("Guess my secret? ");
-  if (answer === 'SeCrEt') {
+  if (answer >= 1 && answer <= 10) {
    console.log("You guessed it!");
    break;
-  }
+ }
+ else {
+   console.log("Answer must be between 1 and 10!");
+ }
 }
```

Comment:
Looks very close. I think that now you're able to test this code, you should be able to push it over the finish line. Essentially, though, since `answer = 0` and since 0 is falsey, your `while` loop may never meet the true condition to run.